### PR TITLE
feat: use pnpm instead of yarn

### PR DESCRIPTION
**Pull request by reednaa**

### CODE REVIEW

**Comment by jsanmigimeno** [comment]
I'm getting the following warning when running the image, not sure if it will cause problems.
image

File: dockerfile.relayer [hunk 1]
```diff
@@ -1,30 +1,33 @@
...
 WORKDIR /usr/catalyst-relayer
 
-# Copy packages and install
-COPY package.json yarn.lock tsconfig*.json ./
+# Copy packages
+COPY package.json pnpm-lock.yaml tsconfig*.json ./
+RUN corepack enable
+
+# Install depends
+from base AS prod-deps
```
Comment (jsanmigimeno) [inline dockerfile.relayer]:
```suggestion
FROM base AS prod-deps
```

**Review by jsanmigimeno — CHANGES_REQUESTED** [review]
- README should be update (yarn installation)

**Comment by reednaa** [comment]
> I'm getting the following warning when running the image, not sure if it will cause problems. image

The bigint error is emitted here: https://www.npmjs.com/package/bigint-buffer?activeTab=code

It is supposedly because the C bindings wasn't compiled correctly. However, I don't understand why the bindings wasn't compiled. Maybe because there is a requirement on the dev dependencies that enable compiling?

I always had that secp256k1 error because of https://github.com/cryptocoinjs/secp256k1-node/issues/173

I will try changing the dockerfile to use prune and see if that helps.

**Comment by reednaa** [comment]
It helps. I will undo the prune removal.

**Review by jsanmigimeno — APPROVED** [review]
Looks good!

**Pull request merged**